<launch>
  <arg name="rate" default="30" />

  <node name="ultra_sonic" pkg="MB1222" type="ultra_sonic.py" output="screen" >
    <param name="rate" value="$(arg rate)" />
  </node>
</launch>